com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -541,6 +541,32 @@ public void AddNetworkPrefab(GameObject prefab)
             }
         }
 
+        public void RemoveNetworkPrefab(GameObject prefab)
```

Comment:
> I'll add tests and in another commit.

why not in this PR?

Reply:
It'll be in this PR, I just haven't done it yet.

Reply:
I'm trying getting the PR up without tests first so people can start reviewing it while I write the tests. Thought it'd be a little more parallelizable that way.